<ui version="4.0" >
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ModPlug Plugin Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Quality</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QGroupBox" name="groupBox" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string>Resolution</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QRadioButton" name="bit16RadioButton" >
            <property name="text" >
             <string>16 bit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="bit8RadioButton" >
            <property name="text" >
             <string>8 bit</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1" colspan="2" >
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string>Channels</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QRadioButton" name="stereoRadioButton" >
            <property name="text" >
             <string>Stereo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="monoRadioButton" >
            <property name="text" >
             <string>Mono (downmix)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2" >
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string>Resampling</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QRadioButton" name="resampNearestRadioButton" >
            <property name="text" >
             <string>Nearest (fastest)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="resampLinearRadioButton" >
            <property name="text" >
             <string>Linear (fast)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="resampSplineRadioButton" >
            <property name="text" >
             <string>Spline (good quality)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="resampPolyphaseRadioButton" >
            <property name="text" >
             <string>8-tap Fir (extremely high quality)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="2" >
        <widget class="QGroupBox" name="groupBox_4" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string>Sampling Rate</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QRadioButton" name="khz48RadioButton" >
            <property name="text" >
             <string>48000 Hz</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="khz44RadioButton" >
            <property name="text" >
             <string>44100 Hz</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="khz22RadioButton" >
            <property name="text" >
             <string>22050 Hz</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="khz11RadioButton" >
            <property name="text" >
             <string>11025 Hz</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Effects</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QGroupBox" name="reverbGroupBox" >
         <property name="title" >
          <string>Reverb</string>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Depth (%)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>10</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2" >
           <widget class="QLabel" name="label_9" >
            <property name="minimumSize" >
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="frameShape" >
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="text" >
             <string>0</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
            <property name="margin" >
             <number>2</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="3" >
           <widget class="QSlider" name="reverbDepthSlider" >
            <property name="maximum" >
             <number>100</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>Delay (ms)</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>10</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="2" >
           <widget class="QLabel" name="label_10" >
            <property name="minimumSize" >
             <size>
              <width>35</width>
              <height>0</height>
             </size>
            </property>
            <property name="frameShape" >
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="text" >
             <string>40</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
            <property name="margin" >
             <number>2</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="3" >
           <widget class="QSlider" name="reverbDelaySlider" >
            <property name="minimum" >
             <number>40</number>
            </property>
            <property name="maximum" >
             <number>200</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QGroupBox" name="bassGroupBox" >
         <property name="title" >
          <string>Bass Boost</string>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label_5" >
            <property name="text" >
             <string>Amount (%)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>10</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2" >
           <widget class="QLabel" name="label_13" >
            <property name="minimumSize" >
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="frameShape" >
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="text" >
             <string>0</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
            <property name="margin" >
             <number>2</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="3" >
           <widget class="QSlider" name="bassAmountSlider" >
            <property name="maximum" >
             <number>100</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="label_6" >
            <property name="text" >
             <string>Range (Hz)</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>10</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="2" >
           <widget class="QLabel" name="label_14" >
            <property name="minimumSize" >
             <size>
              <width>35</width>
              <height>0</height>
             </size>
            </property>
            <property name="frameShape" >
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="text" >
             <string>10</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
            <property name="margin" >
             <number>2</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="3" >
           <widget class="QSlider" name="bassRangeSlider" >
            <property name="minimum" >
             <number>10</number>
            </property>
            <property name="maximum" >
             <number>100</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QGroupBox" name="surGroupBox" >
         <property name="title" >
          <string>Surround</string>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Depth (%)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>10</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2" >
           <widget class="QLabel" name="label_11" >
            <property name="minimumSize" >
             <size>
              <width>35</width>
              <height>0</height>
             </size>
            </property>
            <property name="frameShape" >
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="text" >
             <string>0</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="3" >
           <widget class="QSlider" name="surDepthSlider" >
            <property name="maximum" >
             <number>100</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>Delay (ms)</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>10</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="2" >
           <widget class="QLabel" name="label_12" >
            <property name="minimumSize" >
             <size>
              <width>35</width>
              <height>0</height>
             </size>
            </property>
            <property name="frameShape" >
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="text" >
             <string>5</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="3" >
           <widget class="QSlider" name="surDelaySlider" >
            <property name="minimum" >
             <number>5</number>
            </property>
            <property name="maximum" >
             <number>40</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QGroupBox" name="preampGroupBox" >
         <property name="title" >
          <string>Preamp</string>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label_7" >
            <property name="text" >
             <string>Volume</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>10</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2" >
           <widget class="QLabel" name="preampLabel" >
            <property name="minimumSize" >
             <size>
              <width>35</width>
              <height>0</height>
             </size>
            </property>
            <property name="frameShape" >
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="text" >
             <string>0</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
            <property name="margin" >
             <number>2</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="3" >
           <widget class="QSlider" name="preampSlider" >
            <property name="minimum" >
             <number>-30</number>
            </property>
            <property name="maximum" >
             <number>30</number>
            </property>
            <property name="singleStep" >
             <number>0</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition" >
             <enum>QSlider::TicksBelow</enum>
            </property>
            <property name="tickInterval" >
             <number>30</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3" >
           <widget class="QLabel" name="label_15" >
            <property name="text" >
             <string>Note: Setting the preamp
too high may cause clipping!</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3" >
      <attribute name="title" >
       <string>Misc</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QGroupBox" name="groupBox_9" >
         <property name="title" >
          <string>General</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QCheckBox" name="fileNameCheckBox" >
            <property name="text" >
             <string>Use filename as song title</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QCheckBox" name="noiseCheckBox" >
            <property name="text" >
             <string>Noise reduction</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QCheckBox" name="checkBox_6" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>Fast playlist info</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QCheckBox" name="amigaCheckBox" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>Play Amiga MOD</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_10" >
         <property name="title" >
          <string>Looping</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QRadioButton" name="dontLoopRadioButton" >
            <property name="text" >
             <string>Don't loop</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QRadioButton" name="loopRadioButton" >
            <property name="text" >
             <string>Loop</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QSpinBox" name="loopSpinBox" >
            <property name="maximum" >
             <number>100</number>
            </property>
            <property name="value" >
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2" >
           <widget class="QLabel" name="label_8" >
            <property name="text" >
             <string>time(s)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>10</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0" colspan="2" >
           <widget class="QRadioButton" name="loopForeverRadioButton" >
            <property name="text" >
             <string>Loop forever</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>211</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>reverbDepthSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_9</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>219</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel" >
     <x>199</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reverbDelaySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_10</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>191</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel" >
     <x>219</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>surDepthSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_11</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>200</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>199</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>surDelaySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_12</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>193</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel" >
     <x>187</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bassAmountSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_13</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>408</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel" >
     <x>420</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bassRangeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_14</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>392</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel" >
     <x>397</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>294</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel" >
     <x>170</x>
     <y>371</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
